src/frontends/common/src/frontend_manager.cpp

```diff
@@ -390,6 +390,8 @@ void InputModel::set_tensor_partial_value(Place::Ptr place, const void* min_valu
     FRONT_END_NOT_IMPLEMENTED(set_tensor_partial_value);
 }
 
+InputModel::~InputModel() = default;
```

Comment:
as I see - the method is pure virtual, do we need default implementation then?

Reply:
we can explicitly define and declare destructors in all derived classes. 
`InputModel::~InputModel() = default;` definition for base class helps avoids this.

Reply:
already merged